jck/jck.mk

```diff
@@ -130,13 +144,61 @@ ifeq (8, $(JDK_VERSION))
-   COMPILER_LANG_EXPR_TESTS_GROUP10=$(Q)lang/EXPR/expr463;lang/EXPR/expr497;lang/EXPR/expr490;lang/EXPR/expr232;lang/EXPR/expr340;lang/EXPR/expr172;lang/EXPR/expr516;lang/EXPR/expr724;lang/EXPR/expr186;lang/EXPR/expr529;lang/EXPR/expr181;lang/EXPR/expr175;lang/EXPR/expr347;lang/EXPR/expr378;lang/EXPR/expr385;lang/EXPR/expr143;lang/EXPR/expr371;lang/EXPR/expr715;lang/EXPR/expr527;lang/EXPR/expr188;lang/EXPR/expr518;lang/EXPR/expr520;lang/EXPR/expr712;lang/EXPR/expr376;lang/EXPR/expr144;lang/EXPR/expr382;lang/EXPR/expr349;lang/EXPR/expr746;lang/EXPR/expr110;lang/EXPR/expr322;lang/EXPR/expr128;lang/EXPR/expr325;lang/EXPR/expr117;lang/EXPR/expr748;lang/EXPR/expr777;lang/EXPR/expr313;lang/EXPR/expr121;lang/EXPR/expr119;lang/EXPR/expr126;lang/EXPR/expr314;lang/EXPR/expr383;lang/EXPR/expr713;lang/EXPR/expr377;lang/EXPR/expr348;lang/EXPR/expr370;lang/EXPR/expr142;lang/EXPR/expr526;lang/EXPR/expr384;lang/EXPR/expr189;lang/EXPR/expr346;lang/EXPR/expr174;lang/EXPR/expr180;lang/EXPR/expr379;lang/EXPR/expr187;lang/EXPR/expr173;lang/EXPR/expr341;lang/EXPR/expr517;lang/EXPR/expr528;lang/EXPR/expr118;lang/EXPR/expr315;lang/EXPR/expr127;lang/EXPR/expr749;lang/EXPR/expr120;lang/EXPR/expr312;lang/EXPR/expr129;lang/EXPR/expr116;lang/EXPR/expr324;lang/EXPR/expr747;lang/EXPR/expr323;lang/EXPR/expr111$(Q)
-	
+   COMPILER_LANG_EXPR_TESTS_GROUP10=$(Q)lang/EXPR/expr463;lang/EXPR/expr497;lang/EXPR/expr490;lang/EXPR/expr232;lang/EXPR/expr340;lang/EXPR/expr172;lang/EXPR/expr516;lang/EXPR/expr724;lang/EXPR/expr186;lang/EXPR/expr529;lang/EXPR/expr181;lang/EXPR/expr175;lang/EXPR/expr347;lang/EXPR/expr378;lang/EXPR/expr385;lang/EXPR/expr143;lang/EXPR/expr371;lang/EXPR/expr715;lang/EXPR/expr527;lang/EXPR/expr188;lang/EXPR/expr518;lang/EXPR/expr520;lang/EXPR/expr712;lang/EXPR/expr376;lang/EXPR/expr144;lang/EXPR/expr382;lang/EXPR/expr349;lang/EXPR/expr746;lang/EXPR/expr110;lang/EXPR/expr322;lang/EXPR/expr128;lang/EXPR/expr325;lang/EXPR/expr117;lang/EXPR/expr748;lang/EXPR/expr777;lang/EXPR/expr313;lang/EXPR/expr121;lang/EXPR/expr119$(Q)
+   COMPILER_LANG_EXPR_TESTS_GROUP11=$(Q)lang/EXPR/expr126;lang/EXPR/expr314;lang/EXPR/expr383;lang/EXPR/expr713;lang/EXPR/expr377;lang/EXPR/expr348;lang/EXPR/expr370;lang/EXPR/expr142;lang/EXPR/expr526;lang/EXPR/expr384;lang/EXPR/expr189;lang/EXPR/expr346;lang/EXPR/expr174;lang/EXPR/expr180;lang/EXPR/expr379;lang/EXPR/expr187;lang/EXPR/expr173;lang/EXPR/expr341;lang/EXPR/expr517;lang/EXPR/expr528;lang/EXPR/expr118;lang/EXPR/expr315;lang/EXPR/expr127;lang/EXPR/expr749;lang/EXPR/expr120;lang/EXPR/expr312;lang/EXPR/expr129;lang/EXPR/expr116;lang/EXPR/expr324;lang/EXPR/expr747;lang/EXPR/expr323;lang/EXPR/expr111$(Q)
+
```

Comment:
Will this affect the original test that uses `COMPILER_LANG_EXPR_TESTS_GROUP10`?

Reply:
Thanks for spotting @llxia. Added a new target`jck-compiler-lang-EXPR-group11` under compiler.lang/playlist.xml to run the tests in `COMPILER_LANG_EXPR_TESTS_GROUP11` in regular mode. This is the only new group which impacts the regular targets however. The rest of the new subgroups added in this PR only apply to multijvm targets (e.g. `COMPILER_LANG_CONV_TESTS_GROUP1` is only used in target `jck-compiler-lang-CONV-group1_multijvm`. The corresponding target for regular mode, `jck-compiler-lang-CONV`, does not use the broken down approach but runs everything under lang/CONV.